backend/data/src/main/kotlin/io/tolgee/formats/importCommon/ImportFormat.kt

```diff
@@ -125,6 +125,15 @@ enum class ImportFormat(
     messageConvertorOrNull = appleConvertor,
   ),
 
+  XCSTRINGS(
```

Comment:
```suggestion
  APPLE_XCSTRINGS(
```

To make the import and export format name consistent.
There is an inconsistency between the `APPLE_STRINGSDICT` export format and the `STRINGSDICT` import format, and I can see the current name is probably trying to match that, but let's keep the new format names consistent.